draft-bbs-signatures.md

```diff
@@ -443,11 +439,11 @@ Procedure:
 
 1. (A, e, s) = (octets_to_point(signature.A), OS2IP(signature.e), OS2IP(signature.s))
 
-2. pub_key = octets_to_point(PK)
+2. if subgroup_check(A) is INVALID, return INVALID
 
-3. if subgroup_check(A) is INVALID
+3. if KeyValidate(PK) is INVALID, return INVALID
 
-4. if KeyValidate(pub_key) is INVALID
+4. W = octets_to_point(PK)
```

Comment:
We need to add the key validation check to verify. The Point at infinity will allow any signature to verify. `subgroup_check` won't catch this since the point at infinity is in the subgroup. Should I submit a fix to this or do you want to change it?

Reply:
I don't think this PR removes KeyValidate from verify??

Reply:
I'm not saying it does. I'm saying we should add it.

Reply:
But the function already has it?

Reply:
Should key validate check for infinity and do the subgroup, on curve checks? That would allow a common check. Or maybe we add a function called `IsValidPoint` and use that to perform all the checks and reuse it where necessary

Reply:
Have raised #126 to capture this